tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -248,6 +249,15 @@
             edpm_ovn_remote_probe_interval: 60000
             edpm_ovn_ofctrl_wait_before_clear: 8000
     EOF
+    {{ os_diff_dir }}/os-diff diff {{ os_diff_data_dir }}/tripleo/ovs_external_ids/standalone/ovs_external_ids.json edpm-crd.yaml --crd --service ovs_external_ids -f ${PWD}/{{ os_diff_dir }}/config.yaml
```

Comment:
I'd split up creating the YAML file from the os-diff call, make it 2 separate Ansible shell tasks, to make the output clearer in case of failure.